feat: Update README.md (#3)
virtualidentityag/storyblok-app-starter: README.md

# Storyblok App Starter

> Since Storyblok's offical app starter is still based on Vue 2 / Nuxt 2, you can find an up-2-date app launcher with Vue 3 / Nuxt 3 and a revised oauth authentication here.

Reviewer: ```suggestion
Storyblok's [official app starter is still based on Nuxt 2)[https://github.com/storyblok/storyblok-workflow-app].
This starter provides a Nuxt 3 alternative.
```
